android/src/main/java/com/hieuvp/fingerprint/ReactNativeFingerprintScannerModule.java

```diff
@@ -114,6 +114,11 @@ public void isSensorAvailable(final Promise promise) {
         }
     }
 
+     @ReactMethod
+     public void isFingerPrintIdentifyRefresh() {
```

Comment:
If this function is to reset mFingerprintIdentify then please use relevant name for the function.
isFingerPrintIdentifyRefresh - Doesn't suit as per the functionality.